composer.json

```diff
@@ -16,16 +16,16 @@
         "spryker/kernel": "^3.30.0",
         "spryker/messenger": "^3.0.0",
         "spryker/money": "^2.0.0",
-        "spryker/oms": "^7.0.0 || ^8.0.0 || ^10.0.0 || ^11.0.0",
```

Comment:
this change is bc breaking
you cannot drop BC version support

Reply:
Fixed

Reply:
which validation throws this error?
we cannot remove these version support in a minor release

Reply:
CI / Prefer Lowest is throwing this error

Reply:
If those are untestable, then you do not know if 7 or 8 etc even still work
In that case, removing "impossible" versions and therefore versions with >90% failure rate seems fine.

Are there any known customers using outdated versions and could speak to those actually still working with current changes?
You would have to proof this, e.g. put v7 or v8 into vendor dependency and run tests manually
If that is working out, you can keep the current version map and allow the failure on CI level.

Reply:
I see that minimum required versions are coming from cascading dependencies
And currently, there is no way to get `spryker/sales` installed lower than `11.0.0`, the same comes for `spryker/oms::10.0.0`
But my thoughts are the following:

Dependency adding usually causes major (if we are not talking about edge cases)
But dependency removal could be done in a minor
What if cascading dependency gets rid from `spryker/sales` or `spryker/oms` for example?
This will potentially have an impact on the minimum required version of `spryker/sales` or `spryker/oms`
But our module already dropped support for lower versions in this case

Reply:
@dereuromark, any more thoughts?

Reply:
As I said: you can either fix the other modules or this is fine as is, the important fact is testability as this is the only way to confirm things being compatible.